src/layout/navbar/NavBarLinks/NavLinks.tsx

```diff
@@ -0,0 +1,55 @@
+import { Stack, Tooltip, UnstyledButton, rem } from '@mantine/core';
+import { IconHome2 } from '@tabler/icons-react';
+import classes from '../NavBar.module.css';
+import { mockdata } from 'config/navbarConfig';
+import { useAppSelector } from 'hooks/redux-hooks';
+import { getUser } from 'redux/selectors/userSelectors';
+
+type Props = {
+  icon: typeof IconHome2;
+  label: string;
+  active?: boolean;
+  onClick?(): void;
+};
+
+export const NavbarLink = ({ icon: Icon, label, active, onClick }: Props) => {
+  return (
+    <Tooltip label={label} position="right" transitionProps={{ duration: 0 }}>
+      <UnstyledButton
+        onClick={onClick}
+        className={`${classes.link} ${active ? classes.active : ''}`}
+        data-active={active || undefined}
+      >
+        <Icon style={{ width: rem(20), height: rem(20) }} stroke={1.5} />
+      </UnstyledButton>
+    </Tooltip>
+  );
+};
+
+interface NavbarLinksProps {
+  navigate: (path: string) => void;
+  location: { pathname: string };
+}
```

Comment:
use type Props for components

```suggestion
type Props = {
  navigate: (path: string) => void;
  location: { pathname: string };
}
```